packages/astro/test/content-layer.test.js

```diff
@@ -0,0 +1,89 @@
+import assert from 'node:assert/strict';
+import { before, describe, it } from 'node:test';
+import { loadFixture } from './test-utils.js';
+
+describe('Content Collections', () => {
+	describe('Query', () => {
+		let fixture;
+		before(async () => {
+			fixture = await loadFixture({ root: './fixtures/content-layer/' });
+			await fixture.build();
+		});
+
+		describe('Collection', () => {
```

Comment:
Do you envision so many nested test cases (e.g. `describe`)? If not, we could have 1/2 describe maximum, because running `--test-only` would mean to add `.only` to all the `describe` functions (that's how `node:test` works)

Reply:
Yeah, I grabbed the structure from the content collections test, which has far more cases. For this I do expect there to be more tests, but not so many now to need so many levels